builder/builder.go

```diff
@@ -719,6 +782,29 @@ func (b *Builder) getFullBundleSet(bundles bundleSet) (bundleSet, error) {
 	return set, nil
 }
 
+func populateSetFromPackages(source map[string]bool, dest map[string]*bundle, filename string) error {
```

Comment:
While they're functionally equivalent, the type should be a `bundleSet` for `dest`.

Reply:
Done